SocialCareCaseViewerApi/V1/Factories/ResponseFactory.cs

```diff
@@ -117,5 +118,35 @@ public static WorkerResponse ToResponse(this Domain.Worker worker)
                 Teams = worker.Teams
             };
         }
+
+        public static GetPersonResponse ToResponse(Person person)
```

Comment:
Do we want to be more explicit with the compiler about what can and can't be null.

If we use `#nullable enable` [C# Docs](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references) we can then be explicit if a Type can be also null. For instance we then know person definitely is not null.

Currently we could pass null to this function and would null reference error on line 129 for example.

Reply:
I'll implement these changes in another PR as discussed on Slack